### PR TITLE
Adding Sicilian language check

### PR DESCRIPTION
**Pull request by dpaleino**
Hello,

I work for an association promoting regional and minority languages. In particular, we work with Sicilian (ISO 639-2 code **scn**), and are translating some projects to it. Internally we use Pootle to manage translations.

I created a Sicilian language check, with the most common errors. I tested it on our server, hopefully I didn't forget anything!

Thank you for considering merging this to the main repository.

David

### CODE REVIEW

File: translate/lang/test_scn.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from translate.filters.test_checks import fails, passes
+from translate.lang.ro import SicilianChecker
```
Comment (unho) [inline translate/lang/test_scn.py]: Fix this import.

Comment (dpaleino) [reply translate/lang/test_scn.py]: Done in https://github.com/translate/translate/pull/3919/commits/fc701bc9f4c4187b77250373a630bc6aba71d13b , sorry for the mistake (yes, I used the Romanian-language checks as a reference :) ).

Thank you!
David

**Comment by unho** [comment]
Your changes make some test to fail. Please check Travis log to find the failure and fix it.

**Comment by unho** [comment]
Please squash all commits together. And use a proper commit message.

**Comment by dpaleino** [comment]
@unho I think it's ready, while I was at it I also added one more check with its testsuite. Thank you for your support!

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**